### PR TITLE
fix docker image and secrets validation

### PR DESCRIPTION
**Pull request by GalRabinDemisto**
Fix secrets

### CODE REVIEW

File: Tests/scripts/hook_validations/secrets.py [hunk 1]
```diff
@@ -154,14 +150,14 @@ def search_potential_secrets(secrets_file_paths):
         high_entropy_strings = []
         secrets_found_with_regex = []
         yml_file_contents = None
-        file_path_temp, file_extension = os.path.splitext(file_path)
+        file_path_temp, file_extension = '/'.join(file_path.split('/')[:-1]), os.path.splitext(file_path)[1]
```
Comment (yaakovi) [inline Tests/scripts/hook_validations/secrets.py]: use `os.path.dirname`

File: Tests/scripts/hook_validations/secrets.py [hunk 2]
```diff
@@ -321,6 +329,28 @@ def get_file_contents(file_path, file_extension):
         raise
 
 
+def extract_text_from_md(file_path):
+    with open(file_path, mode='r') as f:
+        html = f.read()
+    soup = BeautifulSoup(html, features="html.parser")
```
Comment (yaakovi) [inline Tests/scripts/hook_validations/secrets.py]: not all md files are HTML

File: Tests/scripts/hook_validations/secrets.py [hunk 3]
```diff
@@ -212,9 +209,9 @@ def create_temp_white_list(file_contents):
     return temp_white_list
 
 
-def retrieve_related_yml(file_path_temp):
+def retrieve_related_yml(file_dir_path_temp):
     matching_yml_file_contents = None
-    yml_file = file_path_temp + '.yml'
+    yml_file = file_dir_path_temp + '/' + file_dir_path_temp.split('/')[-1] + '.yml'
```
Comment (yaakovi) [inline Tests/scripts/hook_validations/secrets.py]:
```suggestion
    yml_file = os.path.join(file_dir_path_temp, os.path.basename(file_dir_path_temp) + '.yml')
```

File: Tests/scripts/hook_validations/secrets.py [hunk 4]
```diff
@@ -155,14 +149,17 @@ def search_potential_secrets(secrets_file_paths):
         secrets_found_with_regex = []
         yml_file_contents = None
         file_path_temp, file_extension = os.path.splitext(file_path)
```
Comment (yaakovi) [inline Tests/scripts/hook_validations/secrets.py]:
```suggestion
         _, file_extension = os.path.splitext(file_path)
```

File: Tests/scripts/hook_validations/secrets.py [hunk 5]
```diff
@@ -155,14 +149,17 @@ def search_potential_secrets(secrets_file_paths):
...
+        file_dir_path_temp = os.path.dirname(file_path)
         skip_secrets = False
 
         secrets_white_list = set(conf_secrets_white_list)
         # get file contents
         file_contents = get_file_contents(file_path, file_extension)
+        # Validate if it is integration file
```
Comment (yaakovi) [inline Tests/scripts/hook_validations/secrets.py]:
```suggestion
        # Validate if it is integration documentation file
```

File: Tests/scripts/hook_validations/secrets.py (hunk 3 above)
Comment (GalRabinDemisto) [reply Tests/scripts/hook_validations/secrets.py]: I just read why it's important to use the os.path module (Thanks)

**Pull request merged**